CHANGELOG.md

```diff
@@ -1 +1,9 @@
+- Enables functions runtime to be set in `firebase.json` (#2241, thanks @quentinvernot!), for example:
```

Comment:
```suggestion
- Enables Firebase Functions runtime to be set in `firebase.json` (#2241, thanks @quentinvernot!), for example:
```

Reply:
Firebase Functions isn't a thing! Cloud Functions for Firebase